components/PageNotFound.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <section>
+    <v-container>
+      <v-stack
+        justify="space-between"
+        md-direction="column-reverse"
+        align="center"
+        md-justify="space-evenly"
+        class="error-content-container"
+      >
+        <v-image
+          path="images/404-illustration.png"
+          class="error-illustration"
+        />
+        <v-stack direction="column" class="error-content">
```

Comment:
This wrapping `v-stack` may not be needed as there's only one child element inside.

Reply:
done. [commit](https://github.com/arcana-network/website/pull/85/commits/adc70e73438069953bad10e762ec4bf064d006a3)